readme.md

```diff
@@ -1,23 +1,440 @@
+
+If wish to only disable Jira-related user interactions coming from Mattermost such as creating issues, you can disable these features by setting **Allow users to connect their Mattermost accounts to Jira** to **false** in **System Console > Plugins > Jira**. You will then need to restart the plugin in **System Console > Plugins > Plugin Management** to update the UI for users currently logged in to Mattermost, or they can refresh to see the changes. This setting does not affect Jira webhook notifications.
+
+#### Why do I get an error `WebHooks can only use standard http and https ports (80 or 443).`?
+
+Jira only allows webhooks to connect to the standard `ports 80 and 443`. If you are using a non-standard port, you will need to set up a proxy between Jira and your Mattermost instance to let Jira communicate over `port 443`.
+
+#### How do I handle credential rotation for the Jira webhook?
+
+Generate a new secret in **System Console > Plugins > Jira**, then paste the new webhook URL in your Jira webhook configuration.
+
+#### What changed in the Jira 2.1 webhook configuration?
+
+In Jira 2.1 there's a modal window for a "Channel Subscription" to Jira issues. This requires a firehose of events to be sent from Jira to Mattermost, and the Jira plugin then "routes" or "drops" the events to particular channels. The Channel Subscription modal \(which you can access by going to a particular channel, then typing `jira /subscribe`\) provides easy access for Mattermost Channel Admins to set up which notifications they want to receive per channel.
+
+If your organization's infrastructure is set up in such a way that your Mattermost instance can't connect to your Jira instance, the Channel Subscriptions feature won't be accessible. Instead, you will need to use the [Legacy Webhooks](admininstrator-guide/notification-management.md#legacy-webhooks) feature supported by the Jira plugin, which allows a Jira webhook to post to a specific channel.
+
 ## License
 
 This repository is licensed under the Apache 2.0 License, except for the [server/enterprise](server/enterprise) directory which is licensed under the [Mattermost Source Available License](LICENSE.enterprise). See [Mattermost Source Available License](https://docs.mattermost.com/overview/faq.html#mattermost-source-available-license) to learn more.
 
 ## Development
 
 Read our [development docs](https://mattermost.gitbook.io/plugin-jira/development/environment) for this project, as well as the [Developer Workflow](https://developers.mattermost.com/extend/plugins/developer-workflow/) and [Developer Setup](https://developers.mattermost.com/extend/plugins/developer-setup/) documentation for more information about developing and extending plugins.
+
+### Environment
+
+To contribute to the project see https://www.mattermost.org/contribute-to-mattermost.
```

Comment:
Wonder if we should remove this? The link is not working and the contribution process maybe changing.